test/long/org/apache/cassandra/cql3/ViewComplexTest.java

```diff
@@ -57,6 +57,9 @@
 @RunWith(Parameterized.class)
 public class ViewComplexTest extends CQLTester
 {
+    @ClassRule
+    public static Timeout classTimeout = new Timeout(20, TimeUnit.MINUTES);
```

Comment:
Isn't 20 too much? I see the long tests have by default 8 minutes timeout, is this one going that far?